lib/membrane/video_compositor/wgpu/wgpu.ex

```diff
@@ -2,18 +2,30 @@ defmodule Membrane.VideoCompositor.Wgpu do
   @moduledoc """
   This module implements video composition in wgpu
   """
```

Comment:
Can't add a comment to the line with `defmodule`, so it's here. I think this module could get a better name than `Wgpu` now, since we don't need to distinguish modules based on the implementation anymore.